typescript/infra/fork.sh

```diff
@@ -27,16 +27,16 @@ done
 # exit 1 on any subsequent failures
 set -e
 
-echo "=== Run checker against forked $ENVIRONMENT ==="
+echo "=== Run $MODULE checker against forked $ENVIRONMENT ==="
```

Comment:
Can we disable these in the gh action until we get them working?

Reply:
wanted to at least test the deployers

Reply:
punting to followup

Reply:
Sgtm